src/main/java/uk/ac/ic/wlgitbridge/bridge/db/postgres/PostgresProjectLock.java

```diff
@@ -0,0 +1,76 @@
+package uk.ac.ic.wlgitbridge.bridge.db.postgres;
+
+import uk.ac.ic.wlgitbridge.bridge.lock.ProjectLock;
+import uk.ac.ic.wlgitbridge.data.LockAllWaiter;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+import java.util.concurrent.locks.ReentrantReadWriteLock;
+
+public class PostgresProjectLock implements ProjectLock {
```

Comment:
Is this intentionally in this PR (vs the next one)?

Reply:
Bah, good catch. `git add -A` is a dangerous tool